charts/elasticsearch-umbrella/charts/elasticsearch-statefulset/templates/service.yaml

```diff
@@ -37,7 +37,14 @@ metadata:
     {{- include "statefulset.labels" . | nindent 4 }}
 spec:
   clusterIP: None
+  {{- if semverCompare ">=7.0.0-0" .Values.image.tag }}
+  publishNotReadyAddresses: true
```

Comment:
Similar to the previous one, for Elastic 7.X it required to publish the service before it's completely ready.